packages/react-composites/src/composites/MeetingComposite/adapter/MeetingAdapter.ts

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* eslint-disable @typescript-eslint/adjacent-overload-signatures */
+
+import {
+  CallAdapterCallManagement,
+  CallAdapterDeviceManagement,
+  CallIdChangedListener,
+  DisplayNameChangedListener,
+  IsMuteChangedListener,
+  IsScreenSharingOnChangedListener,
+  IsSpeakingChangedListener,
+  ParticipantJoinedListener,
+  ParticipantLeftListener
+} from '../../CallComposite';
+import {
+  ChatAdapterThreadManagement,
+  MessageReadListener,
+  MessageReceivedListener,
+  MessageSentListener
+} from '../../ChatComposite';
+import { MeetingState } from '../state/MeetingState';
+
+import type { AdapterState, AdapterDisposal, AdapterPages } from '../../common/adapters';
+
+/**
+ * Page state the Meeting composite could be in.
+ * @alpha
+ */
+export type MeetingCompositePage = 'configuration' | 'meeting' | 'error' | 'errorJoiningTeamsMeeting' | 'removed';
+
+/**
+ * Functionality for managing the current meeting.
+ * @alpha
+ */
+export interface MeetingAdapterMeetingManagement
+  extends Pick<
+      CallAdapterCallManagement,
+      | 'joinCall'
+      | 'leaveCall'
+      | 'startCamera'
+      | 'stopCamera'
+      | 'onToggleCamera'
+      | 'mute'
+      | 'unmute'
+      | 'startCall'
+      | 'startScreenShare'
+      | 'stopScreenShare'
+      | 'createStreamView'
+      | 'disposeStreamView'
+    >,
+    Pick<
+      CallAdapterDeviceManagement,
+      | 'setCamera'
+      | 'setMicrophone'
+      | 'setSpeaker'
+      | 'askDevicePermission'
+      | 'queryCameras'
+      | 'queryMicrophones'
+      | 'querySpeakers'
+    >,
+    Pick<
+      ChatAdapterThreadManagement,
+      'fetchInitialData' | 'sendMessage' | 'sendReadReceipt' | 'sendTypingIndicator' | 'loadPreviousChatMessages'
+    > {
+  removeParticipant(userId: string): Promise<void>;
```

Comment:
It looks like a common part of each adapter, but not sure which interface this should be staying(maybe just keep it here)